### PR TITLE
fix(learn): replace single bactick with <code> tag for Crowdin

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

In the [Use model.findById() to Search Your Database By _id](https://raw.githubusercontent.com/freeCodeCamp/freeCodeCamp/master/curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/use-model.findbyid-to-search-your-database-by-id.md) challenge, Crowdin's parser converted the `>` in the inline code block `Model.findById() -> Person` to an HTML entity `&gt;` which was causing a custom QA check to trigger.

This PR changes the backticks to `<code></code>` and then we escape the `>` with a backslash.  This properly renders on `/learn` still.

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
Looks good to me. Reviewed via video call with Randy.

**Comment by ojeytonwilliams** [comment]
One thing that concerns me is that these Crowdin specific changes are likely to create future headaches for us.  Without context a contributor could easily end up 'fixing' this.

How about leaving an HTML comment `<!-- using code tags for Crowdin -->`?  Or just leaving this for now until we've had a chance to discuss it with Crowdin.

**Comment by raisedadead** [comment]
We are going to be doing a head branch rename. I understand this is a high-priority PR. I can stow this on a separate branch on the main repo for now if you all prefer.

Or, is we have shared the issue with Crowdin we can wait for their response and close this for now.

Let me know.

**Comment by RandellDawson** [comment]
We actually need it to be merged before soft launch.  We were keeping it open in the hopes Crowdin would respond and create a fix, so we did not need to merge it.

**Comment by raisedadead** [comment]
I do not see that Crowdin will deploy any fixes in the next 12 hours, when we launch. If this PR is good to go - please merge.

**Comment by RandellDawson** [comment]
@raisedadead Can I just retarget `main` now?

**Comment by raisedadead** [comment]
I am categorically moving PRs over to main. You should just leave it to me for now.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**
